src/components/Histogram.js

```diff
@@ -12,7 +12,13 @@ export default class Histogram {
     this.chart = vcharts.chart('histogram', {
       el: el,
       values: data,
-      bin: options.bin
+      bin: options.bin,
+      xAxis: {
+        title: options.bin
+      },
+      yAxis: {
+        title: 'Count'
```

Comment:
Generally speaking, we might want to make this configurable. For instance, a particular user of this histogram may want to specify the units of counting in the label, etc.